### DISCUSSION TITLE
Blank Page when trying to go to SearXNG

### DISCUSSION THREAD

**Discussion by eshaane** [searxng/searxng#1482] [Q&A] [answered]
I have recently installed searxng-docker on a Kali machine but when I visit the IP of the server all I see is a blank white page, though when I visit localhost on the computer the search engine does show up, I tried using the port,  changing the port in settings.yml but of no avail. I couldn't find anyone else that had the same issue so I made a new question :).

**Comment by return42** [comment, marked as answer]
Might not related, just to link:

- https://github.com/searxng/searxng/issues/1397

**Reply by eshaane** [reply]
I don't think its related but thanks for the help

**Reply by mrpaulblack** [reply]
If you use caddy it could be that you did not define the hostname of the server correctly. So if you visit the site via localhost it displays search and if you visit the site via the IP of the server you see a blank page by caddy and not SearXNG, since its defined for localhost...

Edit the `.env` file and set the SEARXNG_HOSTNAME var to your IP, does that help?

**Reply by eshaane** [reply]
tysm for the help this solution worked for me
